R/data-population.R

```diff
@@ -104,12 +104,15 @@ calendar_quarter_to_quarter_id <- function(calendar_quarter) {
 #' @rdname quarter_year_labels
 #' @export
 quarter_id_to_calendar_quarter <- function(quarter_id) {
```

Comment:
I think that this will be a much more efficient vectorization than using `vapply()` (and also cleaner to read): 

```
quarter_id_to_calendar_quarter <- function(quarter_id) {

  quarter <- quarter_id %% 4
  quarter[quarter == 0] <- 4
  year <- (quarter_id - quarter)/4 + 1900
  convert_calendar_quarter(year, quarter)
}
```

Reply:
That's way better, thanks!

Reply:
Sorry I've been a naughty and shoved a bit of caching code in here to hopefully again make calls from hintr a bit faster.